includes/admin/class-wcmp-export.php

```diff
@@ -1541,12 +1542,18 @@ private function saveOrderCollection(array $orderIds): array
         $this->orderCollection = (new OrderCollection())->setApiKey($apiKey);
 
         foreach ($orderIds as $orderId) {
-            $wcOrder         = WCX::get_order($orderId);
-            $orderSettings   = new OrderSettings($wcOrder);
-            $deliveryOptions = $orderSettings->getDeliveryOptions();
-
+            $wcOrder                = WCX::get_order($orderId);
+            $orderSettings          = new OrderSettings($wcOrder);
+            $deliveryOptions        = $orderSettings->getDeliveryOptions();
             $labelDescriptionFormat = new LabelDescriptionFormat($wcOrder, $orderSettings, $deliveryOptions);
-
+          
+            $deliveryOptions->getShipmentOptions()->setSignature($orderSettings->hasSignature());
```

Comment:
Als je shipment options ff in een variabele zet kan het volgende blok statements wat korter